### ISSUE TITLE
rename load_regs_ls_wgt_lon_lat and extract_land?

### ISSUE THREAD

**Issue by mathause** [MESMER-group/mesmer#236]
After #235 it might be worth to rename (and move) load_regs_ls_wgt_lon_lat and extract_land. But let's do this in another PR.

**Comment by mathause** [comment]
See also #36

**Comment by mathause** [comment]
Closing this issue. The functions should be removed eventually, so no point in renaming them.
